ext/nnstreamer/tensor_filter/tensor_filter_support_cc.h

```diff
@@ -0,0 +1,83 @@
+/**
+ * GStreamer Tensor_filter, C++ Subplugin Support. (this is not a subplugin)
+ * Copyright (C) 2020 MyungJoo Ham <myungjoo.ham@samsung.com>
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Library General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Library General Public License for more details.
+ *
+ */
+/**
+ * @file	tensor_filter_support_cc.h
+ * @date	15 Jan 2020
+ * @brief	Base class for tensor_filter subplugins of C++ classes.
+ * @see		http://github.com/nnsuite/nnstreamer
+ * @author	MyungJoo Ham <myungjoo.ham@samsung.com>
+ * @bug		No known bugs except for NYI items
+ *
+ * @details
+ *    This is not a subplugin, but a helper for C++ subplugins.
+ *    If you want to write a wrapper for neural network frameworks
+ *    or a hardware adpator in C++, this is what you want.
+ *    If you want to attach a single C++ class/object as a filter
+ *    (a.k.a. custom filter) to a pipeline, use tensor_filter_cpp
+ *
+ * To Packagers:
+ *
+ * This is to be exposed with "nnstreamer-c++-dev"
+ */
+#ifndef __NNS_TENSOR_FILTER_CPP_SUBPLUGIN_SUPPORT_H__
+#define __NNS_TENSOR_FILTER_CPP_SUBPLUGIN_SUPPORT_H__
+#ifdef __cplusplus
+
+#include <nnstreamer_plugin_api_filter.h>
+
```

Comment:
Maybe.. I need to move the class definition to nnstreamer_plugin_api_filter.h

Reply:
Given [this](https://github.com/nnsuite/nnstreamer/pull/2039#issuecomment-581772593), moving is better.